<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1680</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vampire, the eternal deck builder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabEditorModule">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="OngletRechercheCarte">
       <attribute name="title">
        <string>Library search</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </widget>
      <widget class="QWidget" name="OngletRechercheCrypte">
       <attribute name="title">
        <string>Crypt search</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </widget>
      <widget class="QWidget" name="OngletProba">
       <attribute name="title">
        <string>Deck View</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </widget>
      <widget class="QWidget" name="OngletGoldFish">
       <attribute name="title">
        <string>Playground</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1680</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionEnregistrer_le_Deck"/>
    <addaction name="actionExport_Deck"/>
    <addaction name="actionImprimer_le_Deck"/>
    <addaction name="actionNew_Deck"/>
    <addaction name="actionOpen_a_deck"/>
    <addaction name="actionClose_deck"/>
   </widget>
   <widget class="QMenu" name="menuParam_trages">
    <property name="title">
     <string>Param√®trages</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuParam_trages"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEnregistrer_le_Deck">
   <property name="text">
    <string>Enregistrer le Deck</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionImprimer_le_Deck">
   <property name="text">
    <string>Imprimer le Deck</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionExport_Deck">
   <property name="text">
    <string>Export Deck</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionNew_Deck">
   <property name="text">
    <string>New Deck</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_a_deck">
   <property name="text">
    <string>Open a deck</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_deck">
   <property name="text">
    <string>Close deck</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
